src/components/GameItem.js

```diff
@@ -9,17 +9,39 @@ class GameItem extends Component {
     layer: PropTypes.number.isRequired,
   }
 
+  constructor(props) {
+    super(props);
+    this.state = {
+      spottedItem: false	      
+    };
+  }	  
+
+   spottedItem = () => {
+    this.setState({
+      spottedItem: !this.state.spottedItem
+    });
+
+    this.props.itemClick(this.props.type);
+
+     console.log('You have spotted an item!')
+   }	  
+
   render() {
+    let spottedItemStyle = undefined;
+     if (this.state.spottedItem) { 
+      this.props.type === "litter" ? spottedItemStyle = "spotted-litter" : spottedItemStyle = " spotted-nature";
+    }
+
     const itemStyle = {
       bottom: `${this.props.height}px`, // use props.height to offset from the bottom of screen
       zIndex: this.props.layer, // use props.layer to set z-index, so we display ontop of background
     };
 
     // Update this to select the correct icon for each item
-    const icon = ItemIcons.rock;
+    const icon = ItemIcons[this.props.type];
 
     return (
-      <div className="game-item" style={itemStyle}>
+      <div className={`game-item ${spottedItemStyle}`} style={itemStyle} onClick={this.spottedItem}>
```

Comment:
In this context, when I read `onClick = {this.spottedItem}`, the name `spottedItem` read funnily to me. Is that a function? Is that an object?

After a second, it read better to me, but there may be a better name out there for this.